CHANGELOG.md

```diff
@@ -48,7 +48,11 @@
 - Added `httpx[http2]` as dependency ([#1808](https://github.com/neptune-ai/neptune-client/pull/1808))
 - Decreased `batch_size` for `FloatSeries` from 100 to 1 ([#1793](https://github.com/neptune-ai/neptune-client/pull/1793))
 - Introduced `RunCreation` operation ([#1817](https://github.com/neptune-ai/neptune-client/pull/1817))
+<<<<<<< aw/serialize-to-proto
```

Comment:
Rebase went wrong?

Reply:
Omg, I was doing it with the github UI and it apparently didn't catch it - corrcting now